src/SuperimposeMesh/src/SICAD.cpp

```diff
@@ -1288,6 +1288,15 @@ bool SICAD::setProjectionMatrix
 }
 
 
+void SICAD::setOglToCam(const std::vector<float>& ogl_to_cam)
+{
+    if (ogl_to_cam.size() != 4)
+        throw std::runtime_error("ERROR::SICAD::setOglToCam\nERROR:\n\tWrong size provided for ogl_to_cam.\n\tShould be 4, was given " + std::to_string(ogl_to_cam.size()) + ".");
```

Comment:
`Wrong size provided for ogl_to_cam.` -> `Wrong input size.`

Reply:
Done!